icons/CoralTick/index.js

```diff
@@ -0,0 +1 @@
+export { CoralTick } from "./CoralTick";
```

Comment:
You can get rid of this file because you already have a named export in `CoralTick.jsx`